### PR TITLE
[docs]  Update Orbit guide's structure, add Windows installation instructions

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

This PR:
- Add context about Windows (that it is in preview) and installation instructions for Windows (link to release page)
- Also add a highlight about launching updates and link to the relevant doc where we describe the steps
- Organize the current information and divide them into sections.
- Update callout at the end of the page to include Windows instructions too.
- Update doc's description to add "update" part.

# Test Plan

## Preview

CleanShot 2024-04-11 at 00 05 37@2x

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/build/orbit.mdx [hunk 1]
```diff
@@ -1,34 +1,51 @@
...
-description: Accelerate your development workflow with one-click build launches and simulator management.
+description: Accelerate your development workflow with one-click build and update launches and simulator management.
 ---
 
 import { Terminal } from '~/ui/components/Snippet';
 import Video from '~/components/plugins/Video';
+import { Tabs, Tab } from '~/ui/components/Tabs';
 
-Expo Orbit for macOS makes it faster to install and run builds from EAS, local files, or run Snack projects, on simulators and physical devices.
+Expo Orbit for macOS and Windows makes it faster to install and run builds from EAS, local files, or run Snack projects, on simulators and physical devices.
```
Comment (amandeepmittal) [inline docs/pages/build/orbit.mdx:10]:
```suggestion
Expo Orbit for macOS and Windows enables faster to install and launch builds or updates from EAS, local files, or run Snack projects, on simulators and physical devices.
```

Let's also update the intro to add the context about "updates".

File: docs/pages/build/orbit.mdx [hunk 2]
```diff
@@ -1,34 +1,51 @@
...
+import { Tabs, Tab } from '~/ui/components/Tabs';
 
-Expo Orbit for macOS makes it faster to install and run builds from EAS, local files, or run Snack projects, on simulators and physical devices.
+Expo Orbit for macOS and Windows enables faster to install and launch builds or updates from EAS, local files, or run Snack projects, on simulators and physical devices.
 
 <Video file="orbit/basic-features.mp4" />
 
+## Why Orbit
+
 Before Orbit, installing builds from EAS (Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible.
```
Comment (amandeepmittal) [inline docs/pages/build/orbit.mdx:16]:
```suggestion
Before Orbit, installing builds or updates from EAS (Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible.
```

**Review by gabrieldonadel — APPROVED** [review]
Nice, thanks for updating this

File: docs/pages/build/orbit.mdx [hunk 3]
```diff
@@ -1,34 +1,51 @@
...
 
 <Video file="orbit/basic-features.mp4" />
 
-Before Orbit, installing builds from EAS (Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible.
+## Why Orbit
+
+Before Orbit, installing builds or updates from EAS (Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible.
```
Comment (amandeepmittal) [inline docs/pages/build/orbit.mdx:16]:
```suggestion
Before Orbit, installing builds or updates from EAS (on Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible.
```

Fix typo when addressing devices part.

**Review by Simek — APPROVED** [review]

**Pull request merged**